JsonRPC/wox.py

```diff
@@ -4,9 +4,9 @@
 import sys
 import inspect
 
-class Wox(object):
+class Flow.Launcher(object):
```

Comment:
I think this dot still needs to be removed. I wanted to change & push that but it seems that i have no permissions to do it.

Reply:
good catch. Cheers